tools/tree-viz.html

```diff
@@ -17,34 +17,33 @@
 
     <style>
       #out > svg {
-        position: fixed;
-        left: 0;
-        top: 35px;
-        max-width: 100%;
-        max-height: 100%;
+        width: 100%;
+        height: 100vh;
       }
     </style>
   </head>
   <body>
-    <input style="width: 100%" id="in" type="text" value='count_over_time({level="info"}[10s])' />
+    <textarea style="width: 100%; resize: vertical" id="in" type="text">
+count_over_time({level="info"}[10s])</textarea
+    >
     <div id="cursor-pos"></div>
     <div id="out"></div>
     <script type="module">
-      import('@grafana/lezer-logql').then(
```

Comment:
Can we revert these unrelated changes or is there a reason for this?

Reply:
thats my bad, just the prettier extension. ill revert